lib/hammer_cli/validator.rb

```diff
@@ -98,13 +142,17 @@ def all(*to_check)
     end
 
     def option(to_check)
-      all(to_check)
+      OneOptionConstraint.new(@options, to_check)
```

Comment:
This might eventually cause failing tests in various hammer plugins (like here https://github.com/theforeman/hammer-cli-foreman/pull/233/files#diff-dbd90652e9404f49d32e20e0c7f4a976L31 ). I checked `hammer-cli-{foreman,foreman-tasks,katello}` and `hammer_cli_{foreman_remote_execution,foreman_docker}`, those aren't affected by this change